deployment/bin/entrypoint.sh

```diff
@@ -2,17 +2,33 @@
 # if you change this file and the places it looks for local paths:
 # please also update the launch.json for vscode accordingly
 # the things that must be kept in sync are KB_DEPLOYMENT_CONFIG and PYTHONPATH
+#
+# Two main options here:
+# 1. run as `entrypoint.sh` - this starts the staging service webapp
+# 2. run as `entrypoint.sh dts_watcher [--debug]` - this starts only the Data
+#    transfer service file watcher. Optionally include --debug to print
+#    debugging logs.
 
-#top section for local running
+# top section for local running
 DIR="$( cd "$( dirname "$0" )" && pwd )"
-if [ -d "$DIR/../../staging_service" ]; then
-    PYTHONPATH="$DIR/../../staging_service"
+PROJECT_ROOT="$( cd "$DIR/../.." && pwd)"
+
+if [ -d "$PROJECT_ROOT/staging_service" ]; then
+    export PYTHONPATH="$PROJECT_ROOT"
     export KB_DEPLOYMENT_CONFIG="$DIR/../conf/local.cfg"
+    WATCHER_PATH="$PROJECT_ROOT/scripts/run_dts_watcher.py"
+fi
+
+# bottom section for running inside docker
+if [ -d "/kb/deployment/lib/staging_service" ]; then
+    export PYTHONPATH="/kb/deployment/lib"
+    WATCHER_PATH="/kb/deployment/scripts/run_dts_watcher.py"
+    # environment variable for KB_DEPLOYMENT_CONFIG is expected to be set
+    # when the container is started
 fi
 
-#bottom section for running inside docker
-if [ -d "kb/deployment/lib/staging_service" ]; then
-    PYTHONPATH="kb/deployment/lib/staging_service"
-    # environment variable for KB_DEPLOYMENT_CONFIG set in docker-compose.yml
+if [ "$1" == "dts_watcher" ]; then
+    python3 "$WATCHER_PATH" $2
```

Comment:
From my understanding it would be a better practice to have one entrypoint.sh for the webapp and the other for the watching thread. Can we split it up?

Reply:
Sure. I can do that. Does that mean separate Dockerfiles? That's the main reason I did it that way.

Reply:
No, we can still use one Dockerfile. We can override the entrypoint in the deployment or in the docker-compose.

Reply:
Ok, I'll revert this it and make a new entrypoint that sets the PYTHONPATH before running the watcher script.

Reply:
Ok, done. I split it out into two entrypoints and updated the Dockerfile and docker-compose.yml files accordingly.

Reply:
awesome